helloextend-protection/admin/class-helloextend-protection-admin.php

```
@@ -172,7 +172,8 @@ public function enqueue_scripts()
             wp_enqueue_script($this->helloextend_protection, plugin_dir_url(__FILE__) . 'js/helloextend-protection-admin.js', array('jquery'), $this->version, false);
         }
 
-        $js_file_version = filemtime(plugin_dir_url(__FILE__) . 'js/helloextend-protection-remove-ignored-category.js');
+        $js_path = plugin_dir_path(__FILE__) . 'js/helloextend-protection-remove-ignored-category.js';
```

PHPMD (2.15.0)

The variable $js_path is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)